README.md

```diff
@@ -12,20 +12,24 @@ User requirements
 
 2. Carbonite console 8.3.0.293+
 
-3. Carbonite Migrate — The source server must have Carbonite Migrate installed and licensed on it.
+3. Carbonite Migrate — the machine you will be migrating (source) **must** have Carbonite Migrate software installed and licensed on it. This machine also needs to be added to the Carbonite console application on your Carbonite server:
```

Comment:
Uppercase console.  Just to confirm, Carbonite Console is the name of the product?

Reply:
Changed to Carbonite Replication Console.